src/porepy/numerics/ad/discretizations.py

```diff
@@ -489,42 +491,48 @@ def bound_pressure_face(
 
     def _flux_function(
         self,
-        perm_argument: pp.ad.Ad_array,
         potential: pp.ad.Ad_array,
     ) -> pp.ad.Ad_array:
-        """
+        """Flux from potential, BCs and vector sources.
 
-        Args:
-            perm_function: pp.ad.Function returning permeability given the perm_argument.
-            perm_argument: Ad_array representing the variable upon which perm_function depends.
-            potential: Potential for the flux law: flux=-K\grad(potential - vector_source)
+        Parameters:
+            perm_function: pp.ad.Function returning permeability given a list of
+                subdomains.
+            perm_argument: Ad_array representing the variable upon which perm_function
+                depends.
+            potential: Potential for the flux law
+                :math:`flux=-K\nabla(potential - vector_source)`.
 
         Returns:
+            Ad_array representing the flux.
+
+        ..note:
```

Comment:
On a matter of style, I am not sure this note should be part of the docstring (it concerns implementation details rather than something a user may want to know).

Also, I think the directive is not to put something under Returns in the docstring.